crates/sui-core/src/authority/authority_store.rs

```diff
@@ -621,24 +641,34 @@ impl AuthorityStore {
 
     /// Checks if the input object identified by the InputKey exists, with support for non-system
     /// packages i.e. when version is None.
-    pub fn multi_input_objects_exist(
+    pub fn multi_input_objects_available(
```

Comment:
Renaming from exist to available doesn't really make it obvious what is now different. Please document and maybe find a better name.

Reply:
updated